iaso/management/commands/delete_accounts.py

```diff
@@ -422,7 +422,7 @@ def handle(self, *args, **options):
         print(Device.objects.filter(projects=None).delete())
 
         for f in forms_without_projects:
-            print(OrgUnitType.objects.filter(reference_form=f).update(reference_form=None))
+            print(OrgUnitType.reference_forms.through.objects.filter(form=f).delete())
```

Comment:
These modifications are probably superfluous since the deletion will cascade to m2m by default.

Reply:
Why add them, then?

Reply:
Because without unit tests, it makes me feel better :)